src/main/scala/Cqrs/Aggregate.scala

```diff
@@ -23,9 +28,9 @@ object Aggregate {
   final case class VersionedEvents[E](version: Int, events: List[E])
 
   sealed trait EventDatabaseOp[E, A]
-  final case class ReadAggregateExistence[E](id: AggregateId) extends EventDatabaseOp[E, Error Xor Boolean]
-  final case class ReadAggregate[E](id: AggregateId, fromVersion: Int) extends EventDatabaseOp[E, Error Xor List[VersionedEvents[E]]]
-  final case class AppendAggregateEvents[E](id: AggregateId, events: VersionedEvents[E]) extends EventDatabaseOp[E, Error Xor Unit]
+  final case class ReadAggregateExistence[E](tag: Tag, id: AggregateId) extends EventDatabaseOp[E, Error Xor Boolean]
```

Comment:
for this, there is currently a separate Database method as defined by the typeclass - https://github.com/svalaskevicius/eventflow/blob/multiEventsProjection/src/main/scala/Cqrs/Database.scala#L12

its not available in the aggregate actions as no aggregate should ever need that.. I think :)